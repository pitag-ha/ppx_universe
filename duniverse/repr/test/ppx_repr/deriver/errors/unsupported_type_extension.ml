type t = [%typ] [@@deriving repr]
