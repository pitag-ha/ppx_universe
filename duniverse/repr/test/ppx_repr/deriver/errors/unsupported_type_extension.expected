File "unsupported_type_extension.ml", line 1, characters 9-15:
Error: ppx_repr: unprocessed extension [%typ ] encountered.
